src/System.Private.ServiceModel/tests/Common/src/project.json

```diff
@@ -1,55 +1,10 @@
 {
   "dependencies": {
-    "System.Collections": "4.0.10-beta-*",
-    "System.Collections.Concurrent": "4.0.10-beta-*",
-    "System.Collections.NonGeneric": "4.0.0-beta-*",
-    "System.Collections.Specialized": "4.0.0-beta-*",
-    "System.ComponentModel": "4.0.0-beta-*",
-    "System.ComponentModel.EventBasedAsync": "4.0.10-beta-*",
-    "System.Diagnostics.Contracts": "4.0.0-beta-*",
-    "System.Diagnostics.Debug": "4.0.10-beta-*",
-    "System.Diagnostics.Tools": "4.0.0-beta-*",
-    "System.Globalization": "4.0.10-beta-*",
-    "System.IO": "4.0.10-beta-*",
-    "System.IO.Compression": "4.0.0-beta-*",
-    "System.Linq": "4.0.0-beta-*",
-    "System.Linq.Expressions": "4.0.10-beta-*",
-    "System.Linq.Queryable": "4.0.0-beta-*",
-    "System.Net.Http": "4.0.0-beta-*",
-    "System.Net.NameResolution": "4.0.0-beta-*",
-    "System.Net.Primitives": "4.0.10-beta-*",
-    "System.Net.Security": "4.0.0-beta-*",
-    "System.Net.Sockets": "4.0.10-beta-*",
```

Comment:
When I run OuterLoop tests, I see several failures due to not being able to find System.Net.Sockets or System.Reflection.Emit, etc.  I don't think we can remove the explicit dependencies on all the other packages. Did you find you needed to do this to build, or did you believe deleting these was a good cleanup?